python/ccf/ledger.py

```diff
@@ -375,56 +439,25 @@ def get_size():
         )
 
 
-class Transaction:
-    """
-    A transaction represents one entry in the CCF ledger.
-    """
-
+class Entry(abc.ABC):
     _file: Optional[BinaryIO] = None
     _header: TransactionHeader
     _public_domain_size: int = 0
-    _next_offset: int = LEDGER_HEADER_SIZE
     _public_domain: Optional[PublicDomain] = None
     _file_size: int = 0
     gcm_header: Optional[GcmHeader] = None
-    _tx_offset: int = 0
-    _ledger_validator: LedgerValidator
 
-    def __init__(self, filename: str, ledger_validator: LedgerValidator):
-        self._ledger_validator = ledger_validator
+    @abc.abstractmethod
```

Comment:
An implemented abstractmethod is... weird?

Reply:
Yeah, I've remove the use of the `abc` module.